impeller/renderer/backend/gles/vertex_descriptor_gles.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <map>
+#include <vector>
+
+#include "flutter/fml/macros.h"
+#include "impeller/renderer/backend/gles/gles.h"
+#include "impeller/renderer/backend/gles/proc_table_gles.h"
+#include "impeller/renderer/command.h"
+#include "impeller/renderer/vertex_descriptor.h"
+
+namespace impeller {
+
+class VertexDescriptorGLES {
```

Comment:
This was modeled after VertexDescriptorMTL but is way more than just that. Rename class and dcoument.

Reply:
Renamed to BufferBindingsGLES